qxf2_scheduler/candidates.py

```diff
@@ -83,7 +84,7 @@ def add_candidate(job_role):
             #getting the unique candidate id for new candidate
             candidate_id = Candidates.query.filter(Candidates.candidate_email==candidate_email).value(Candidates.candidate_id)
             #storing the candidate id and job id in jobcandidate table
-            add_job_candidate_object = Jobcandidate(candidate_id=candidate_id,job_id=job_id,url='')
+            add_job_candidate_object = Jobcandidate(candidate_id=candidate_id,job_id=job_id,url='',candidate_status='Interview Scheduled')
```

Comment:
@monoranjanqxf2  why are you hardcoding the status while reading the candidates. The candidate status should change after the schedule operation am I correct?